pymatviz/ptable.py

```diff
@@ -636,6 +651,7 @@ def ptable_heatmap(
     value_font_size: int = 12,
     tile_size: float | tuple[float, float] = 0.9,
     rare_earth_voffset: float = 0.5,
```

Comment:
maybe rename `rare_earth_voffset` to `f_block_voffset` for consistency? don't imagine this kwarg is used a lot so could just make it a breaking rename. or use 

```py
if "rare_earth_voffset" in kwargs:
    f_block_voffset = kwargs.pop("rare_earth_voffset")
    print("deprecation warning: use f_block_voffset instead of rare_earth_voffset
```

during a migration period

Reply:
Thanks for proposing this... Maybe a breaking change? As I don't assume plotters to be integrated into automatic workflows much?